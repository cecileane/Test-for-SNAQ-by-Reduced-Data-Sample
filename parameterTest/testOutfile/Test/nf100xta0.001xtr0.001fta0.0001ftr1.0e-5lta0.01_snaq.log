optimization of topology, BL and inheritance probabilities using:
 hmax = 1,
 tolerance parameters: ftolRel=1.0e-5, ftolAbs=0.0001,
                       xtolAbs=0.001, xtolRel=0.001.
 max number of failed proposals = 100, liktolAbs = 0.01.
rootname for files: nf100xta0.001xtr0.001fta0.0001ftr1.0e-5lta0.01_snaq
BEGIN: 2 runs on starting tree (2,3,(4,(5,(1,6):0.386):1.19):0.028);

main seed 7384
seed: 7384 for run 1
Sat Oct  1 17:22:39 2016

 BEGIN SNaQ for run 1, seed 7384 and hmax 1 changed starting topology by NNI move

Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED for not having more moves to propose: movesfail [0,25,25,4,0,61], Nmov [217,25,25,4,10000,61]
END optTopLevel: found minimizer topology at step 161 (failures: 99) with -loglik=155.49626 and ht_min=[0.12573,0.33535,0.55437,0.32983]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 3		 3		 1	 |	 NA 		 NA 		 NA 
mvorigin	 36		 36		 3	 |	 1		 1		 --
mvtarget	 39		 39		 1	 |	 1		 1		 --
chdir		 10		 10		 1	 |	 3		 3		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 72		 54		 3	 |	 0		 0		 --
Total		 160		 142		 9	 |	 5		 5		 3
Proportion	 -- 		 1.0		 0.1	 |	 -- 		 1.0		 0.6

 FINISHED SNaQ for run 1, -loglik of best 155.49625734558535
(3,(6,5):0.33534748072307535,((1,(2)#H7:::0.6269753504770771):2.1558623651866706,(4,#H7:::0.3730246495229229):2.1558623651866706):0.12573272466778337);
---------------------
seed: 116 for run 2
Sat Oct  1 17:22:46 2016

 BEGIN SNaQ for run 2, seed 116 and hmax 1
Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED for not having more moves to propose: movesfail [0,25,25,4,0,61], Nmov [217,25,25,4,10000,61]
END optTopLevel: found minimizer topology at step 151 (failures: 87) with -loglik=158.13875 and ht_min=[0.32676,0.29504,0.04619,10.0,1.10715]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 1		 1		 1	 |	 NA 		 NA 		 NA 
mvorigin	 35		 35		 4	 |	 0		 0		 --
mvtarget	 34		 34		 3	 |	 1		 1		 --
chdir		 8		 8		 0	 |	 2		 2		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 72		 42		 0	 |	 4		 4		 --
Total		 150		 120		 8	 |	 7		 7		 6
Proportion	 -- 		 1.0		 0.1	 |	 -- 		 1.0		 0.9

 FINISHED SNaQ for run 2, -loglik of best 158.13874671890707
((5,6):0.295036855377915,(((2,#H7:::0.3267586387983987):10.0,1):1.1071489888652724,3):0.04619028306168724,(4)#H7:::0.6732413612016013);
---------------------

Sat Oct  1 17:22:51 2016
MaxNet is (3,(6,5):0.33534748072307535,((1,(2)#H7:::0.6269753504770771):2.1558623651866706,(4,#H7:::0.3730246495229229):2.1558623651866706):0.12573272466778337); 
with -loglik 155.49625734558535
