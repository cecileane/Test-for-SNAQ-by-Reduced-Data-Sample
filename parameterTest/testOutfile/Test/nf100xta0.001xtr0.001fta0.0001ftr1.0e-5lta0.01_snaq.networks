(3,(6,5):0.33534748072307535,((1,(2)#H7:::0.6269753504770771):2.1558623651866706,(4,#H7:::0.3730246495229229):2.1558623651866706):0.12573272466778337);, with -loglik 155.49625734558535 (best network found, remaining sorted by log-pseudolik; the smaller, the better)
(3,(6,5):0.32228833117641603,((1,(#H7:::0.31752002144716784,2):9.085570976511265):1.1126705656717892,(4)#H7:::0.6824799785528322):0.0);, with -loglik 158.6261788083417
(3,(6,5):0.3238917800982085,(#H7:::0.47931187095003447,(4,(2,(1)#H7:::0.5206881290499655):4.776391590558733):0.5727193349039431):0.0);, with -loglik 170.8844112108996
(1,#H7:0.9764784435629488::0.19001739496271108,((4,((3,(6,5):0.3933526089871665):0.0)#H7:0.8337338284480938::0.8099826050372889):0.9790926269195855,2):9.871827790805684);, with -loglik 178.7851426537969
