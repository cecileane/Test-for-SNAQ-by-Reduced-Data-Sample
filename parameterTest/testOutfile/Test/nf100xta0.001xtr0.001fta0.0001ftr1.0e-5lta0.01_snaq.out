(3,(6,5):0.33534748072307535,((1,(2)#H7:::0.6269753504770771):2.1558623651866706,(4,#H7:::0.3730246495229229):2.1558623651866706):0.12573272466778337); -Ploglik = 155.49625734558535
 Dendroscope: (3,(6,5):0.33534748072307535,((1,(2)#H7):2.1558623651866706,(4,#H7):2.1558623651866706):0.12573272466778337);
 Elapsed time: 12.441035848 seconds in 2 successful runs
-------
List of estimated networks for all runs (sorted by log-pseudolik; the smaller, the better):
 (3,(6,5):0.33534748072307535,((1,(2)#H7:::0.6269753504770771):2.1558623651866706,(4,#H7:::0.3730246495229229):2.1558623651866706):0.12573272466778337);, with -loglik 155.49625734558535
 ((5,6):0.295036855377915,(((2,#H7:::0.3267586387983987):10.0,1):1.1071489888652724,3):0.04619028306168724,(4)#H7:::0.6732413612016013);, with -loglik 158.13874671890707
-------
